### PR TITLE
Request dev branch

### PR DESCRIPTION
**Pull request by TahleeJ**
I deleted unnecessary code and improved the formatting of existing to code to be more readable. I also added more styling to my comments, mainly the font-sizes, and I also changed my images to be modal images that enlarge when clicked. I altered my index.html and script.js files to be able to handle the changing slides and their respective modal images when clicked.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (cshei) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: I don't know if this is possible, but can you limit the number of results returned when constructing the query above?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (cshei) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Remove comment?

**Review by cshei — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (TahleeJ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: I'm not quite sure what you mean by that. When the get request is sent to this servlet, I have it set up so that only the specified number of comments will be added to an ArrayList and then returned (as long as numComments is greater than 0, additional comments will be added to the resulting list). Is there something different from what I've done that you're thinking of?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (TahleeJ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Got it!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (cshei) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: Right now, you're running the query and then picking out the first numComments results.  I was just curious whether it was possible to specify such a limit in the query itself, so that "results" only contains at most numComments entries when you call datastore.prepare(query).

Looks like it is possible if you append a ".Limit(numComments)" to line 42 above: https://cloud.google.com/datastore/docs/concepts/queries#cursors_limits_and_offsets

Comment (TahleeJ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: Oh, I see now! I'll add it in, thank you for the new information!

Comment (TahleeJ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: I tried adding in the changes for the query limit, but I'm running into a lot of problems. The page that you linked was for the Google Cloud datastore and the datastore that the walkthrough has us work with is for the App Engine datastore. When I tried making the changes to the Query according to its API within Google Cloud, I ran into problems with the import statements because it kept saying that the package for Cloud's datastore did not exist. When I tried adding in the dependencies in the pom file, it's giving me errors that I'm not sure how to resolve, such as specifying the correct version and scope even though I copied the dependencies from an official page: https://cloud.google.com/datastore/docs/reference/libraries. Should I keep trying to incorporate the limit into my query or should I revert it back to its original form?

Comment (cshei) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: In that case, it's probably not worth the effort... I think what you had originally is fine then.  I thought there might be an easy way to do this with a simple change to the query.

Comment (TahleeJ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:48]: I made reverted the changes and removed the unnecessary comment.

**Comment by TahleeJ** [comment]
Am I missing anything else?

**Comment by ArunChauhan** [comment]
It looks fine to me, but please wait for approval from Chun-Yu since he has looked at your code more closely.

**Review by cshei — APPROVED** [review]

**Pull request merged**
